.github/workflows/automation.yml

```
@@ -51,14 +51,14 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: scala-bionic
-      run: docker build . --file ./scala/Dockerfile.bionic --tag scala-bionic:$(date +%s)
+      run: docker build . --file ./scala/build/Dockerfile.bionic --tag scala-bionic:$(date +%s)
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:74: Quote this to prevent word splitting

(shellcheck)